OCPBUGS-49379 autoAssign=true present in ipaddresspool CR assigns the first available host IP from a network (#94990)
openshift/openshift-docs: modules/nw-metallb-addresspool-cr.adoc

|`spec.autoAssign`
|`boolean`
|Optional: Specifies whether MetalLB automatically assigns IP addresses from this pool.
Specify `false` if you want to explicitly request an IP address from this pool with the `metallb.io/address-pool` annotation.
The default value is `true`.

[NOTE]
====

> For IP address pool configurations, ensure the addresses field specifies only IPs that are available and not in use by other network devices, especially gateway addresses, to prevent conflicts when `autoAssign` is enabled.

Reviewer: ```suggestion
If you enabled the `autoAssign` setting, ensure for IP address pool configurations that the `addresses` field specifies only IP addresses that are available and not in use by other network devices, with care especially taken for gateway addresses.
```
